painttiming.bs

```diff
@@ -56,9 +56,9 @@ This is formally defined as the when <a href="https://html.spec.whatwg.org/multi
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the browser is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
-<dfn export>First Paint</dfn> entry contains a DOMHighResTimeStamp reporting the time when the browser first rendered after navigation. This excludes the default background paint, but includes non-default background paint. This is the first key moment developers care about in page load – when the browser has started to render the page.
+<dfn export>First Paint</dfn> entry contains a DOMHighResTimeStamp reporting the time when the browser first rendered after navigation. This excludes the default background paint, but includes non-default background paint and iframes (regardless of the contents of the iframe). This is the first key moment developers care about in page load – when the browser has started to render the page.
```

Comment:
is this really needed? Feels like it raises some questions.
So either remove this OR make it more clear (enclosing box render for iframe)

Reply:
Currently, the spec talks about the PaintTiming entries as if the concept was page-specific. However, in reality it is a PerformanceEntry associated to a Window/Worker, so I think that giving the iframe example is useful to keep that in mind?